src/main/java/de/tum/in/www1/artemis/service/DataExportService.java

```diff
@@ -0,0 +1,567 @@
+                }
+
+                createResultsTxtFile(submission, exerciseDir);
+            }
+        }
+    }
+
+    private void createQuizAnswersExport(QuizExercise quizExercise, StudentParticipation participation, Path outputDir) throws IOException {
+        Set<QuizQuestion> quizQuestions = quizQuestionRepository.getQuizQuestionsByExerciseId(quizExercise.getId());
+        QuizSubmission quizSubmission;
+
+        for (var submission : participation.getSubmissions()) {
+            quizSubmission = quizSubmissionRepository.findWithEagerSubmittedAnswersById(submission.getId());
+            List<String> multipleChoiceQuestionsSubmissions = new ArrayList<>();
+            List<String> shortAnswerQuestionsSubmissions = new ArrayList<>();
+            for (var question : quizQuestions) {
+                var submittedAnswer = quizSubmission.getSubmittedAnswerForQuestion(question);
+                // if this question wasn't answered, the submitted answer is null
+                if (submittedAnswer != null) {
+                    if (submittedAnswer instanceof DragAndDropSubmittedAnswer dragAndDropSubmittedAnswer) {
+                        dragAndDropQuizAnswerConversionService.convertDragAndDropQuizAnswerAndStoreAsPdf(dragAndDropSubmittedAnswer, outputDir);
+                    }
+                    else if (submittedAnswer instanceof ShortAnswerSubmittedAnswer shortAnswerSubmittedAnswer) {
+                        shortAnswerQuestionsSubmissions.add(createExportForShortAnswerQuestion(shortAnswerSubmittedAnswer));
+                    }
+                    else if (submittedAnswer instanceof MultipleChoiceSubmittedAnswer multipleChoiceSubmittedAnswer) {
+                        multipleChoiceQuestionsSubmissions.add(createExportForMultipleChoiceAnswerQuestion(multipleChoiceSubmittedAnswer));
+                    }
+                }
+            }
+            if (!multipleChoiceQuestionsSubmissions.isEmpty()) {
+                Files.write(outputDir.resolve("quiz_submission_" + submission.getId() + "_multiple_choice_questions_answers.txt"), multipleChoiceQuestionsSubmissions);
+            }
+            if (!shortAnswerQuestionsSubmissions.isEmpty()) {
+                Files.write(outputDir.resolve("quiz_submission_" + submission.getId() + "_short_answer_questions_answers.txt"), shortAnswerQuestionsSubmissions);
+            }
+        }
+
+    }
+
+    private String createExportForMultipleChoiceAnswerQuestion(MultipleChoiceSubmittedAnswer multipleChoiceSubmittedAnswer) {
+        StringBuilder stringBuilder = new StringBuilder();
+        MultipleChoiceQuestion question = (MultipleChoiceQuestion) multipleChoiceSubmittedAnswer.getQuizQuestion();
+        if (question.isSingleChoice()) {
+            stringBuilder.append("Single Choice Question: ");
+        }
+        else {
+            stringBuilder.append("Multiple Choice Question: ");
+        }
+        stringBuilder.append(question.getTitle()).append("\n");
```

Comment:
We've had some issues in the past with the line separator. Would it be better to use the system line separator instead of always \n?